tests/integration/test_hyper_transformer.py

```diff
@@ -83,110 +85,60 @@ def _reverse_transform(self, data):
         return out
 
 
-def get_input_data_with_nan():
-    data = pd.DataFrame({
-        'integer': [1, 2, 1, 3, 1],
-        'float': [0.1, 0.2, 0.1, np.nan, 0.1],
-        'categorical': ['a', 'a', np.nan, 'b', 'a'],
-        'bool': [False, np.nan, False, True, False],
-        'datetime': [
-            np.nan, '2010-02-01', '2010-01-01', '2010-02-01', '2010-01-01'
-        ],
-        'names': ['Jon', 'Arya', 'Arya', 'Jon', 'Jon'],
-    })
-    data['datetime'] = pd.to_datetime(data['datetime'])
-
-    return data
-
-
-def get_input_data_without_nan(index=None):
+def get_input_data():
+    index = [4, 6, 3, 8, 'a', 1.0, 2.0, 3.0]
+    datetimes = pd.to_datetime([
+        np.nan,
+        '2010-02-01',
+        '2010-01-01',
+        '2010-01-01',
+        '2010-01-01',
+        '2010-02-01',
+        '2010-01-01',
+        '2010-01-01',
+    ])
     data = pd.DataFrame({
-        'integer': [1, 2, 1, 3],
-        'float': [0.1, 0.2, 0.1, 0.1],
-        'categorical': ['a', 'a', 'b', 'a'],
-        'bool': [False, False, True, False],
-        'datetime': ['2010-02-01', '2010-01-01', '2010-02-01', '2010-02-01'],
-        'names': ['Jon', 'Arya', 'Jon', 'Jon'],
-    })
-    data['datetime'] = pd.to_datetime(data['datetime'])
-    if index:
-        data.index = index
+        'integer': [1, 2, 1, 3, 1, 4, 2, 3],
+        'float': [0.1, 0.2, 0.1, np.nan, 0.1, 0.4, np.nan, 0.3],
+        'categorical': ['a', 'a', np.nan, 'b', 'a', 'b', 'a', 'a'],
+        'bool': [False, np.nan, False, True, False, True, True, False],
+        'datetime': datetimes,
+        'names': ['Jon', 'Arya', 'Arya', 'Jon', 'Jon', 'Sansa', 'Jon', 'Jon'],
+    }, index=index)
 
     return data
 
 
-def get_reversed(index=None):
-    reverse_transformed = get_input_data_without_nan(index)
-    reverse_transformed['bool'] = reverse_transformed['bool'].astype('O')
-    return reverse_transformed
-
-
-def get_transformed_data(index=None):
-    transformed = pd.DataFrame({
-        'integer.value': [1, 2, 1, 3],
-        'float.value': [0.1, 0.2, 0.1, 0.1],
-        'categorical.value': [0.375, 0.375, 0.875, 0.375],
-        'bool.value': [0.0, 0.0, 1.0, 0.0],
-        'datetime.value': [
-            1.2649824e+18,
-            1.262304e+18,
-            1.2649824e+18,
-            1.2649824e+18
-        ],
-        'names.value': [0.375, 0.875, 0.375, 0.375]
-    })
-    if index:
-        transformed.index = index
-
-    return transformed
-
-
-def get_transformed_nan_data():
+def get_transformed_data():
+    index = [4, 6, 3, 8, 'a', 1.0, 2.0, 3.0]
```

Comment:
Since this is the same in both the input data and transformed, should we pull it out?